pacsman/dicom_interface.py

```diff
@@ -0,0 +1,74 @@
+
+from abc import ABC, abstractmethod
+from collections import namedtuple
+
+
+PatientInfo = namedtuple('PatientInfo', ['first_name', 'last_name', 'patient_id', 'dob',
+                                          'study_ids', 'most_recent_study'])
+
+SeriesInfo = namedtuple('SeriesInfo', ['series_id', 'acquisition_datetime', 'description', 'modality',
+                                       'num_images'])
+
+
+class DicomInterface(ABC):
```

Comment:
I think that making `DicomInterface` and `ABC` means subclasses need to implement all of the `abstract methods`.  Is it possible there will be subclasses that only implement some of them?

Reply:
We probably need them all to be implemented at least for `pynetdicom` and `dcmtk`. But maybe other libraries don't have all the features necessary to implement them all (like send to PACS). For now I think it's a good construct.